assets/i18n/language_pl.properties

```diff
@@ -116,6 +116,12 @@ randomEvent.sportTournament.eventAppearancePopupDescription =  Niedługo, bo {{d
 randomEvent.sportTournament.eventStartPopupDescription = W naszym mieście rozpoczął się turniej sportowy! Będzie on trwał {{noDays}} dni. Spodziewaj się większej ilości klientów.
 randomEvent.sportTournament.calendarDescription = Turniej sportowy będzie trwał {{noDays}} dni. Przygotuj się na większą liczbę klientów w tym okresie.
 
+# RandomPermanentEvent
+
+randomPermanentEvent.buildingCost.title = Zmiana Kosztów Budowy
+randomPermanentEvent.buildingCost.positiveEventAppearancePopupDescription = W związku ze spadającymi kosztami materiałów budowniczych, koszty budowy maleją o {{modifierValueInPercent}} procent!
```

Comment:
Dla mnie lepiej brzmi "ze spadającymi cenami materiałów budowlanych"

Reply:
for me potato tomato